tests/util/test_unit_system.py

```diff
@@ -388,72 +388,41 @@ def test_get_unit_system_invalid(key: str) -> None:
         _ = get_unit_system(key)
 
 
+# fmt: off
 @pytest.mark.parametrize(
     "unit_system, device_class, original_unit, state_unit",
     (
+        # Test distance conversion
         (METRIC_SYSTEM, SensorDeviceClass.DISTANCE, LENGTH_FEET, LENGTH_METERS),
         (METRIC_SYSTEM, SensorDeviceClass.DISTANCE, LENGTH_INCHES, LENGTH_MILLIMETERS),
         (METRIC_SYSTEM, SensorDeviceClass.DISTANCE, LENGTH_MILES, LENGTH_KILOMETERS),
         (METRIC_SYSTEM, SensorDeviceClass.DISTANCE, LENGTH_YARD, LENGTH_METERS),
         (METRIC_SYSTEM, SensorDeviceClass.DISTANCE, LENGTH_KILOMETERS, None),
         (METRIC_SYSTEM, SensorDeviceClass.DISTANCE, "very_long", None),
         # Test speed conversion
-        (
-            METRIC_SYSTEM,
-            SensorDeviceClass.SPEED,
-            SPEED_FEET_PER_SECOND,
-            SPEED_KILOMETERS_PER_HOUR,
-        ),
-        (
-            METRIC_SYSTEM,
-            SensorDeviceClass.SPEED,
-            SPEED_MILES_PER_HOUR,
-            SPEED_KILOMETERS_PER_HOUR,
-        ),
+        (METRIC_SYSTEM, SensorDeviceClass.SPEED, SPEED_FEET_PER_SECOND, SPEED_KILOMETERS_PER_HOUR),
```

Comment:
Alternatively add a comma after the last item for all tuples to have them all be multiline-formatted.

Reply:
agreed

Reply:
I don't think having all multiline really helps with clarity.
Maybe it is better to split the METRIC and US_CUSTOMARY tests?